lichess-bot.py

```diff
@@ -1021,7 +1025,7 @@ def start_lichess_bot() -> None:
 
     max_retries = CONFIG.engine.online_moves.max_retries
     check_python_version()
-    li = lichess.Lichess(CONFIG.token, CONFIG.url, __version__, logging_level, max_retries)
+    li = lichess_communicator(CONFIG.token, CONFIG.url, __version__, logging_level, max_retries)
```

Comment:
With the way this PR has been written, defining `lichess_communicator` isn't needed anymore. `lichess.Lichess` can be used directly here.